tools/development/gen_mediapipe_libs.sh

```diff
@@ -4,20 +4,24 @@
 if [[ -z "${MEDIAPIPE_HOME}" ]]; then
   echo 'ERROR: Define MEDIAPIPE_HOME First!'
 else
-  # for the external objects
-  for ext_obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/external/ -name '*.o' ); do
-    ext_obj_arr+=($ext_obj)
-  done
-  $(gcc ${ext_obj_arr[@]} -shared -o libmediapipe_external.so -fPIC)
-  $(mv libmediapipe_external.so $1)
+  if [ -d "$1" ]; then
+    # `for the external objects
+    for ext_obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/external/ -name '*.o' ); do
+      ext_obj_arr+=($ext_obj)
+    done
+    $(gcc ${ext_obj_arr[@]} -shared -o libmediapipe_external.so -fPIC)
+    $(mv libmediapipe_external.so $1)
 
-  # for the internal objects
-  for int_obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/mediapipe/ -name '*.o' ); do
-    # remove object files which contain main functions
-    if [[ $(nm -Ca $int_obj) != *"T main"* ]]; then
-      int_obj_arr+=($int_obj)
-    fi
-  done
-  $(gcc ${int_obj_arr[@]} -shared -o libmediapipe_internal.so -fPIC)
-  $(mv libmediapipe_internal.so $1)
+    # for the internal objects
+    for int_obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/mediapipe/ -name '*.o' ); do
+      # remove object files which contain main functions
+      if [[ $(nm -Ca $int_obj) != *"T main"* ]]; then
```

Comment:
In order to provide portability, define variables for gcc and nm (e.g., CC/NM). ?= could be helpful.

Reply:
I added like below.
```bash
CC=$(which gcc)
NM=$(which nm)
```
if there is better way, please let me know :)

Reply:
s/=/?=/g